### DISCUSSION TITLE
Managing PR

### DISCUSSION THREAD

**Discussion by kod-man** [vercel/turborepo#7896] [Help]
### Summary

Hi all. Let's say we have 3 applications in our mono repo. 

Each application is being developed by a different team.

I want something like this:

If **Team1** opens pr for their project let's call it **App1**, **Team2**, and **Team3** don't see the pr and don't get a notification, etc

Any idea is welcome

**Comment by anthonyshew** [comment]
Can you explain what this has to do with Turborepo?

**Reply by kod-man** [reply]
It is a mono repo containing several applications. I'm just trying to get an overview. I would like to know if there is a solution for every application in terms of PR management.
